app/services/concerns/course/user_invitation_service/email_invitation_concern.rb

```diff
@@ -161,7 +161,8 @@ def add_existing_users(users)
       if course_user
         existing_course_users << course_user
       else
-        new_course_users << @current_course.course_users.build(user: user[:user], name: user[:name],
+        role = user[:role] || Course::UserInvitation.roles[:student]
```

Comment:
Can you update the documentation for `invite_users`, in that the `:role` key might be expected in the `users` object?